modular_lumos/code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,675 @@
+
+/obj/effect/slime_rune/cerulean
+	colour = "cerulean"
+
+	var/linked_projection
+
+/obj/effect/slime_rune/cerulean/Destroy(force)
+	if(linked_projection)
+		qdel(linked_projection)
+	. = ..()
+
+/obj/effect/cerulean_projection
+	density = FALSE
+	anchored = TRUE
+	var/obj/effect/slime_rune/cerulean/linked_rune
+
+/obj/effect/cerulean_projection/Destroy(force)
+	if(linked_rune?.linked_projection == src)
+		linked_rune.linked_projection = null
+	. = ..()
+	
+/obj/effect/slime_rune/cerulean/Crossed(atom/movable/AM, oldloc)
+	. = ..()
+	if(linked_projection)
+		qdel(linked_projection)
+	var/turf/src_turf = get_turf(src)
+	if(ishuman(AM))
+		var/mob/living/carbon/human/AMHuman = AM
+		var/mob/living/carbon/human/newHuman = new /mob/living/carbon/human(src_turf)
+		newHuman.real_name = AMHuman.real_name
+		AMHuman.dna.transfer_identity(newHuman)
+		newHuman.updateappearance(mutcolor_update=1)
+		newHuman.alpha = 222
+		newHuman.color = COLOR_BLUE_LIGHT
+		newHuman.move_resist = MOVE_FORCE_OVERPOWERING
+		newHuman.a_intent = INTENT_HELP
+		newHuman.density = FALSE
+		newHuman.dir = AMHuman.dir
+		linked_projection = newHuman
+		return
+	else if(isliving(AM) && !ishuman(AM))
+		var/obj/effect/cerulean_projection/newProjection = new /obj/effect/cerulean_projection(src_turf)
+		newProjection.icon = AM.icon
+		newProjection.icon_state = AM.icon_state
+		newProjection.name = AM.name
+		newProjection.desc = AM.desc
+		newProjection.alpha = 222
+		newProjection.color = COLOR_BLUE_LIGHT
+		newProjection.dir = AM.dir
+		newProjection.linked_rune = src
+		linked_projection = newProjection
+
+/obj/effect/slime_rune/pyrite
+	colour = "pyrite"
+	
+	var/list/random_color_list = list("#00aedb","#a200ff","#f47835","#d41243","#d11141","#00b159","#00aedb","#f37735","#ffc425","#008744","#0057e7","#d62d20","#ffa700")
+
+/obj/effect/slime_rune/pyrite/Crossed(atom/movable/AM, oldloc)
+	. = ..()
+	AM.add_atom_colour(pick(random_color_list), WASHABLE_COLOUR_PRIORITY)
+
+/obj/effect/slime_rune/red
+	colour = "red"
+
+/obj/effect/slime_rune/green
+	colour = "green"
+	process_rune = TRUE
+	var/timer = 0
+
+/obj/effect/slime_rune/green/process()
+	if(timer > world.time)
+		return
+	timer = world.time + 10 SECONDS
+
+	var/turf/src_turf = get_turf(src)
+	for(var/obj/item/stack/sheet/mineral/plasma/choosePlasma in src_turf)
+		if(!choosePlasma.use(4))
+			return
+		new /obj/item/stack/sheet/resin(src_turf)
+
+/obj/effect/slime_rune/pink
+	colour = "pink"
+
+/obj/effect/slime_rune/gold
+	colour = "gold"
+
+	var/coin_amount = 0
+
+/obj/effect/slime_rune/gold/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/coin))
+		var/obj/item/coin/chosenCoin = I
+		coin_amount += chosenCoin.value
+		qdel(chosenCoin)
+	else
+		..()
+
+/obj/effect/slime_rune/gold/attack_hand(mob/living/user)
+	var/turf/src_turf = get_turf(src)
+	if(coin_amount < 50)
+		to_chat(user, "You need more coin traveller")
+		return
+	coin_amount -= 50
+	var/obj/item/spawnItem = pick(subtypesof(/obj/item))
```

Comment:
fuck off

Reply:
find me a better way then.

Reply:
im not about to drop a lot of my time to create a white/black list when I worked a lot of time on just the code itself